performance-storage-service/pss_project/api/models/rest/metrics/OLTPBenchMetrics.py

```diff
@@ -1,7 +1,8 @@
 from pss_project.api.models.rest.metrics.LatencyMetrics import LatencyMetrics
 
 class OLTPBenchMetrics(object):
-    def __init__(self, throughput, latency=None):
+    def __init__(self, throughput, incremental_metrics, latency=None):
         self.throughput = throughput
         if latency:
-            self.latency = LatencyMetrics(**latency)
\ No newline at end of file
+            self.latency = LatencyMetrics(**latency)
+        self.incremental_metrics = [dict(metric) for metric in incremental_metrics]
```

Comment:
I think we probably want an `if incremental_metrics:` cause that can be None. Also, it might be nice to create an IncrementalMetrics class. We could probably abstract a parent class from these two Metrics classes to make things easier to update.

Reply:
Done.